### DISCUSSION TITLE
Cannot find the file.

### DISCUSSION THREAD

**Discussion by okikosuke** [pyg-team/pytorch_geometric#4952] [Q&A]
https://github.com/pyg-team/pytorch_geometric/blob/master/examples/argva_node_clustering.py
I ran the above site.

```
NameError                                 Traceback (most recent call last)
[<ipython-input-12-960b11262918>](https://localhost:8080/#) in <module>()
     22                       split_labels=True, add_negative_train_samples=False),
     23 ])
---> 24 path = osp.join(osp.dirname(osp.realpath(__file__)), '..', 'data', 'Planetoid')
     25 dataset = Planetoid(path, name='Cora', transform=transform)
     26 train_data, val_data, test_data = dataset[0]

NameError: name '__file__' is not defined
```

However, I got this error.
Perhaps it is due to the lack of the original data.
Where is the original data?

**Comment by Padarn** [comment]
This is intended to be run as a script and won't work inside `IPython`.

However you could just define this:
```
__file__ = 'path to the file on your local machine'
```

**Comment by okikosuke** [comment]
Thank you for your comment.
Do you have a dataset that could be used as a sample?
If so, would you be willing to share it with us?
Also, for example, if the file is "test.csv" and it is on the desktop, would I just specify the following?

````
__file__ = "C:/Users/***/OneDrive/Desktop/test.csv"
```

**Reply by Padarn** [reply]
The __file__ here is referring to the file itself (the one where this code was written)..it is only used to determine where the dataset will be downloaded to, not to point to your own dataset

**Reply by okikosuke** [reply]
Thank you very much. I had misunderstood, but now I understand.
https://github.com/pyg-team/pytorch_geometric/blob/master/examples/argva_node_clustering.py
Do you have any sample data to run the above code I asked about? If so, please let me know.

**Reply by Padarn** [reply]
The code you shared will download the `Cora` dataset for this task:
```
dataset = Planetoid(path, name='Cora', transform=transform)
train_data, val_data, test_data = dataset[0]
```
